src/IFramePage.php

```diff
@@ -46,52 +47,40 @@ class IFramePage extends Page
-                ->setEmptyString('')
-                ->setDescription(
-                    'Avoids mixed content warnings when iframe content is just available under a specific protocol'
-                ),
-            'Metadata'
-        );
-        $fields->addFieldsToTab('Root.Main', [
-            CheckboxField::create('AutoHeight', 'Auto height (only works with same domain URLs)'),
-            CheckboxField::create('AutoWidth', 'Auto width (100% of the available space)'),
-            NumericField::create('FixedHeight', 'Fixed height (in pixels)'),
-            NumericField::create('FixedWidth', 'Fixed width (in pixels)'),
-            HtmlEditorField::create('Content', 'Content (appears above iframe)'),
-            HtmlEditorField::create('BottomContent', 'Content (appears below iframe)'),
-            HtmlEditorField::create('AlternateContent', 'Alternate Content (appears when user has iframes disabled)')
-        ]);
-
-        // Move the Metadata field to last position, but make a check for it's
-        // existence first.
-        //
-        // See https://github.com/silverstripe-labs/silverstripe-iframe/issues/18
-        $mainTab = $fields->findOrMakeTab('Root.Main');
-        $mainTabFields = $mainTab->FieldList();
-        $metaDataField = $mainTabFields->fieldByName('Metadata');
-        if ($metaDataField) {
-            $mainTabFields->removeByName('Metadata');
-            $mainTabFields->push($metaDataField);
-        }
-        return $fields;
+        $this->beforeUpdateCMSFields(function (FieldList $fields) {
+            $fields->replaceField(
+                'IFrameURL',
+                TextField::create('IFrameURL', $this->fieldLabel('IFrameURL'))
+                ->setRightTitle(
+                    DBField::create_field(
+                        'HTMLText',
+                        'Can be absolute (<em>http://silverstripe.com</em>) '
+                        . 'or relative to this site (<em>about-us</em>).'
+                    )
+                )
+            );
+            $fields->dataFieldByName('IFrameTitle')
+                ->setDescription(_t(__CLASS__ . '.TITLE_DESCRIPTION', 'Used by screen readers'));
+            $fields->replaceField(
+                'ForceProtocol',
+                DropdownField::create('ForceProtocol', $this->fieldLabel('ForceProtocol'))
+                    ->setSource(array('http://' => 'http://', 'https://' => 'https://'))
+                    ->setEmptyString('')
+                    ->setDescription(
+                        'Avoids mixed content warnings when iframe content is just available under a specific protocol'
+                    )
+            );
+
+            $contentField = $fields->dataFieldByName('Content');
+            if ($contentField) {
+                $fields->removeByName('Content');
+                $contentField->setTitle(_t(__CLASS__ . '.db_Content', 'Content (appears above iframe)'));
```

Comment:
`fieldLabel()` only takes the class where the field is defined into account, not the class where it's being called from, so we can't just rely on `$this->fieldLabel('Content')` here like we normally would.